Source/SwiftLintCore/Extensions/Configuration+RulesMode.swift

```diff
@@ -26,6 +26,10 @@ public extension Configuration {
         /// Only enable the rules explicitly listed.
         case only(Set<String>)
 
+        /// Only enable the rule explicitly listed on the command line. The rule may have multiple identifiers,
+        /// hence why this is represented as a Set
```

Comment:
I've changed this one and the one above to clarify the difference between `onlyRules` and `onlyRule`

```
        /// Only enable the rules explicitly listed in the configuration files.
        case only(Set<String>)

        /// Only enable the rule explicitly listed on the command line (and it's aliases).
        case onlyRule(Set<String>)
```

It would be nice if the naming reflected that a bit more, but I don't have any good suggestions.